test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [OPTIONAL] It is required that this file should be ignored by git. It should not appear on your repo. Kindly follow this [link](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files) to correct it.